.github/workflows/release-docker-github-experimental.yml

```
@@ -179,9 +179,8 @@ jobs:
 
       - name: Upload Sentry Sourcemaps
         uses: ./.github/actions/upload-sentry-sourcemaps
-        continue-on-error: true
         with:
           docker_image: ${{ needs.build.outputs.DOCKER_IMAGE }}
           release_version: ${{ needs.build.outputs.RELEASE_VERSION }}
           sentry_auth_token: ${{ secrets.SENTRY_AUTH_TOKEN }}
-          environment: staging
+          environment: staging
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)